examples/src/routes/_list.js

```diff
@@ -39,17 +40,23 @@ export const advancedUseCasesRoutes = [
     { path: '/advanced/chart-configuration', title: 'Chart Configuration', Component: ChartConfiguration }
 ];
 
+export const visualizationUseCasesRoutes = [
+    { path: '/visualization/visualization-by-uri', title: 'Visualization By Uri', Component: VisualizationByUri },
+    { path: '/visualization/visualization-by-identifier', title: 'Visualization By Identifier', Component: VisualizationByIdentifier }
```

Comment:
Just a small note: I would name the titles as `Visualization by URI` and `Visualization by identifier` (note the letter cases).

Reply:
Maybe you can use these titles in <h1> titles in the "pages" themselves.